persistence/src/test/java/com/alibaba/nacos/persistence/configuration/condition/ConfigChangeConfigsTest.java

```diff
@@ -1,98 +0,0 @@
-/*
- * Copyright 1999-2018 Alibaba Group Holding Ltd.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.alibaba.nacos.config.server.configuration;
-
-import com.alibaba.nacos.common.event.ServerConfigChangeEvent;
-import com.alibaba.nacos.plugin.config.constants.ConfigChangeConstants;
-import com.alibaba.nacos.sys.env.EnvUtil;
-import org.junit.Before;
-import org.junit.Test;
-import org.springframework.mock.env.MockEnvironment;
-
-import java.util.Properties;
-
-import static org.junit.Assert.assertEquals;
-
-public class ConfigChangeConfigsTest {
-
-    private static final boolean TEST_WEBHOOK_ENABLED = false;
-
```

Comment:
why remove the UT case ?

Reply:
Yeah, it's a truely confusion. I don't know why I can not run this UT properly. I can not build the whole job with this UT. So I deleted it :(